### PR TITLE
Continue Last Entry intent

### LINKED ISSUE #3392: Siri: Add suggested intent to continue last entry
Now that we are implementing suggested intents (#3388) I think we should add another one to continue whatever the latest entry is.

### PR DESCRIPTION
**Pull request by khoi**
## What's this?
Adds a new intent that continue whatever last time entry is, and is suggested by default. 

### Relationships
Closes #3392

## How is it done?
- fetch the list of timeentries from the API, get the first one.
- create a new TE based on the fetched last TE 

### Side effects
None

## Review hints
None.

## :squid: Permissions
🎙

### CODE REVIEW

**Comment by Odrakir** [comment]
It works 👌 and code looks fine.

Only thing I'm not really happy with is texts. Let's ask Leez to review them again, to me "Continue timer" sounds a little weird...

File: Toggl.Daneel.SiriExtension.UI/IntentViewController.cs [hunk 1]
```diff
@@ -84,6 +84,22 @@ protected IntentViewController(IntPtr handle) : base(handle)
...
+                    {
+                        if (interaction.IntentResponse is ContinueTimerIntentResponse response)
+                        {
+                            desiredSize = showStartTimerSuccess(response.EntryDescription);
+                        }
+                    }
+
+                    if (interaction.IntentHandlingStatus == INIntentHandlingStatus.Ready)
+                    {
+                        desiredSize = showConfirmation("Continue timer?");
```
Comment (Odrakir) [inline Toggl.Daneel.SiriExtension.UI/IntentViewController.cs]: I think this is supposed to be "Start tracking time?" or "Start tracking {entryDescription}?"

Comment (khoi) [reply Toggl.Daneel.SiriExtension.UI/IntentViewController.cs]: I haven't found a way to pass information from `ConfirmContinueTimer` to `HandleContinueTimer` yet. So we don't have `entryDescription` here, same goes for the StopTimerIntent.

I've created an issue so we can look to it later here https://github.com/toggl/mobileapp/issues/4588

Comment (Odrakir) [reply Toggl.Daneel.SiriExtension.UI/IntentViewController.cs]: For now then, let's change the text to "Continue tracking time?"

File: Toggl.Daneel.ExtensionKit/Toggl.Daneel.ExtensionKit.csproj [hunk 2]
```diff
@@ -65,15 +65,13 @@
...
-  <ItemGroup>
-    <BundleResource Include="..\Toggl.Daneel.IntentCodeGen\Toggl.Daneel.IntentCodeGen\en.lproj\Intents.strings">
-      <Link>en.lproj\Intents.strings</Link>
-    </BundleResource>
-  </ItemGroup>
   <ItemGroup>
     <Content Include="..\Toggl.Daneel.IntentCodeGen\Toggl.Daneel.IntentCodeGen\Base.lproj\Intents.intentdefinition">
       <Link>Base.lproj\Intents.intentdefinition</Link>
     </Content>
+    <Content Include="..\Toggl.Daneel.IntentCodeGen\Toggl.Daneel.IntentCodeGen\en.lproj\Intents.strings">
```
Comment (khoi) [inline Toggl.Daneel.ExtensionKit/Toggl.Daneel.ExtensionKit.csproj]: Why the change? `BundleResource` has been changed to `Content`?
